libbeat/common/flowhash/communityid_fips.go

```diff
@@ -19,16 +19,16 @@
 
 package flowhash
 
-var CommunityID = NewCommunityID()
+var CommunityID = newEmptyCommunityID()
 
-type communityIDHasher struct{}
+type emptyCommunityIDHasher struct{}
 
 // NewCommunityID allows to instantiate a flow hasher with custom settings.
-func NewCommunityID() Hasher {
-	return &communityIDHasher{}
+func newEmptyCommunityID() Hasher {
+	return &emptyCommunityIDHasher{}
 }
 
 // Hash returns the hash for the given flow.
-func (h *communityIDHasher) Hash(flow Flow) string {
+func (h *emptyCommunityIDHasher) Hash(flow Flow) string {
```

Comment:
I know the original code was already like this, but as this is a noop implementation, the comment/docs should state that.

Reply:
Changing the signature to `func (h *emptyCommunityIDHasher) Hash(Flow) string` would help there.